crates/cxx-qt-lib/src/core/qtlogging.rs

```diff
@@ -36,6 +36,18 @@ mod ffi {
         /// Outputs a message in the Qt message handler.
         fn qt_message_output(msgType: QtMsgType, context: &QMessageLogContext, string: &QString);
 
+        /// Generates a formatted string out of the type, context, str arguments.
+        #[cxx_name = "qFormatLogMessage"]
+        fn format_log_message(
```

Comment:
Both of these new functions are exported at the top-level of cxx-qt-lib as `cxx_qt_lib::format_log_message;` and `cxx_qt_lib::set_message_pattern` respectively.
Within that namespace, we usually stick to the names provided by Qt with only a case conversion.
So please name these methods `q_format_log_message` and `q_set_message_pattern` to indicate they are coming from Qt directly and aren't e.g. cxx-qt-lib specific functions.